CRM/Batch/Form/Entry.php

```diff
@@ -132,15 +143,14 @@ public function buildQuickForm() {
     // get the profile information
     if ($this->_batchInfo['type_id'] == $batchTypes['Contribution']) {
       CRM_Utils_System::setTitle(ts('Batch Data Entry for Contributions'));
-      $customFields = CRM_Core_BAO_CustomField::getFields('Contribution');
```

Comment:
not used

Reply:
FWIW, to my eye, the handling of `$customFields` in `processMembership()` looks fairly similar (i.e. no obvious/meaningful reads, only obvious writes). Not saying that needs to be addressed at the same time - but just in case you're interested.

Maybe after the `r-test` build finishes, then we just to do 5-min `r-run` on the autobuild site to confirm that the custom-fields still render+save on the batch-entry screen.